lib/helpers/contentfulEntry.js

```diff
@@ -36,21 +37,59 @@ function parseContentfulEntry(contentfulEntry) {
   return Promise.resolve(contentfulEntry);
 }
 
+/**
+ * @param {Object} contentfulEntry
+ * @return {Object}
+ */
+function getSummaryFromContentfulEntry(contentfulEntry) {
+  const result = {
+    id: contentful.getContentfulIdFromContentfulEntry(contentfulEntry),
+    name: contentful.getNameTextFromContentfulEntry(contentfulEntry),
+    type: contentful.getContentTypeFromContentfulEntry(contentfulEntry),
+    createdAt: contentfulEntry.sys.createdAt,
+    updatedAt: contentfulEntry.sys.updatedAt,
+  };
+  logger.debug('parseNameInfoFromContentfulEntry', { result });
+  return result;
+}
+
 /**
  * @param {Object} contentfulEntry
  * @param {String} templateName
  * @return {Object}
  */
 function getMessageTemplateFromContentfulEntryAndTemplateName(contentfulEntry, templateName) {
+  const topicRef = contentfulEntry.fields.topic;
+  const topic = topicRef ? { id: contentful.getContentfulIdFromContentfulEntry(topicRef) } : null;
```

Comment:
2cents in this line: One thing I'm wondering is if sending an empty `{}` is more practical. This way other functions that expect properties to be set inside the `topic`, like `topic.id`, wouldn't have to first check for existence ` result.topic ? topic.id : null`. Also probably helps with unwanted `Cannot read property 'id' of null`.

Reply:
Good call! Will update